### ISSUE TITLE
4.4.0 CLI no longer seems to support -Dproperty=value syntax for substitution

### ISSUE THREAD

**Issue by Gama11** [liquibase/liquibase#1907]
We have a liquibase command that looks something like this:

```sh
/liquibase/liquibase \
       --driver=org.postgresql.Driver \
       --changeLogFile="db.changelog-master.xml" \
       --url=<redacted> \
       --username=<redacted> \
       --password=<redacted> \
       update \
       -Dcoreschema="${core_schema}"
```

After upgrading the CLI from 4.3.3 to 4.4.0, it fails with the following error:

```
Unexpected argument(s): -Dcoreschema=public

For detailed help, try 'liquibase --help' or 'liquibase <command-name> --help'
```

Since the release notes state "this release should be 100% backwards compatible", I expect this is an oversight? Or is there a replacement for the `-Dproperty=value` syntax? I haven't found one looking through `--help` and the [property substitution docs](https://docs.liquibase.com/concepts/basic/changelog-property-substitution.html) at least.

The syntax can also still be found on https://docs.liquibase.com/tools-integrations/cli/home.html:

![image](https://user-images.githubusercontent.com/2620907/122090597-36d63400-ce08-11eb-92e8-d320cd5c5baa.png)

**Comment by mariochampion** [comment]
hey @Gama11  -- thanks for the report! we'll look into this right away.

**Comment by nvoxland** [comment]
Yes, that is an oversite. Lots of pieces to re-implement with the new CLI and that got missed, sorry. We will get it fixed up for 4.4.1.

In the meantime, changelog properties can be set in a few different ways:
- Using the -D syntax as a JAVA_OPTS env variable. Like `set JAVA_OPTS=-Dengine=myisam` and then running `liquibase update` 
- Adding it to the liquibase.properties file as `liquibase.parameter.NAME` like `liquibase.parameter.engine=myisam`

**Comment by nvoxland** [comment]
The changelog properties can also be read from environment variables as well. Like `export ENGINE=myisam`

**Comment by nvoxland** [comment]
I have a fix for this with #1943 

Part of the reason this got missed is because we are working toward a standardizing the arguments for the different interfaces (CLI, maven, gradle, etc.) and the "-Dparam=value" style is something that only fits in the CLI.

While we'll bring back the support for -D arguments in the CLI, we're also marking it as deprecated in favor of setting the parameters in ways that can be consistently used across the different interfaces like the defaults file, environment variables, system properties, etc. 

We may change our mind as we work through the other integrations and the standardization process, but still going to try to shift people to other options for now.
